test/e2e/connectivity_test.go

```diff
@@ -333,28 +368,35 @@ func TestOVSRestartSameNode(t *testing.T) {
 // still works.
 func TestOVSFlowReplay(t *testing.T) {
 	skipIfProviderIs(t, "kind", "stopping OVS daemons create connectivity issues")
+	skipIfHasWindowsNodes(t)
+
 	data, err := setupTest(t)
 	if err != nil {
 		t.Fatalf("Error when setting up test: %v", err)
 	}
 	defer teardownTest(t, data)
 
 	numPods := 2
-	podNames := make([]string, numPods)
-	for idx := range podNames {
-		podNames[idx] = randName(fmt.Sprintf("test-pod-%d-", idx))
+	podInfos := make([]podInfo, numPods)
+	for i := range podInfos {
+		podInfos[i].name = randName(fmt.Sprintf("test-pod-%d-", i))
 	}
 	workerNode := workerNodeName(1)
 
+	if len(clusterInfo.windowsNodes) != 0 {
+		workerNode = workerNodeName(clusterInfo.windowsNodes[0])
+	}
```

Comment:
is this change intentional, you are already skipping the test with `skipIfHasWindowsNodes`?